### ISSUE TITLE
Indicate that rows in the error and matched statuses can be expanded

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#405]
## Overview

Add a visual cue that prompts the user that they can click a row for more information about the match or the error.

### Is your feature request related to a problem? Please describe.

>For those entires which don't process and receive an error message, It is not clear that a user can click on the entry to see the reason for the error. In fact the links on the other entries may lead the user to believe there is no option to click there

### Describe the solution you'd like

Looking for suggestions from design.

#### Error
![2019-03-26 14 04 00](https://user-images.githubusercontent.com/17363/55033284-0a883000-4fd0-11e9-9a71-d6ee7dbcae48.gif)

#### Matched
![2019-03-26 14 03 38](https://user-images.githubusercontent.com/17363/55033297-107e1100-4fd0-11e9-82cf-0ec6435a01db.gif)

**Comment by jwalgran** [comment]
@designmatty How would you suggest changing the styling or content of rows to indicate that they can be expanded. Do you think a traditional "disclosure triangle" enough?

**Comment by designmatty** [comment]
yeah maybe just displaying a down arrow/triangle in the row will suffice for now.

**Comment by dboyer** [comment]
Perhaps complete as part of #482
